plugins/imrelp/imrelp.c

```diff
@@ -882,6 +882,10 @@ ENDafterRun
 
 BEGINmodExit
 CODESTARTmodExit
+	struct sigaction newAct;
```

Comment:
It looks like this struct needs to be static. If so, we do not get the valgrind violations. I haven't checked the API semantics yet, but thought I let you know @Alix82

Reply:
Could be because I don't memset newAct, will update the PR, thanks.

Reply:
> Could be because I don't memset newAct, will update the PR, thanks.

yes, I just went a bit further. The mem init is missing, thus the issue. When static, the loader zeros out.

Reply:
Updated the PR, thank you.